src/test/java/codeu/controller/ActivityServletTest.java

```diff
@@ -26,10 +26,10 @@ public void setup() throws IOException {
     .thenReturn(mockRequestDispatcher);
   }
 
-  @Test
-  public void testDoGet() throws IOException, ServletException {
-	  activityServlet.doGet(mockRequest, mockResponse);
-
-    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
-  }
+//  @Test
+//  public void testDoGet() throws IOException, ServletException {
+//	  activityServlet.doGet(mockRequest, mockResponse);
+//
+//    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
```

Comment:
A little worrisome that this fails, actually.  Submit, and I'll see if I can fix it.

Reply:
Sorry for the late reply but I does fail. That is why I commented it out

Reply:
yup -- I can check it out once you've merged (all looks good to me)